nbs/examples/NonNegativeReconciliation.ipynb

```diff
@@ -5,7 +5,7 @@
    "id": "843cf8de-d678-4243-a8af-d78439058e6a",
```

Comment:
<p>This is a good example of introducing S and tags. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/hierarchicalforecast/pull/102/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>